vendor/github.com/code-ready/machine/drivers/virtualbox/virtualbox.go

```diff
@@ -124,8 +122,8 @@ func (d *Driver) GetCreateFlags() []mcnflag.Flag {
 		mcnflag.StringFlag{
 			Name:   "virtualbox-bundlepath-url",
 			Usage:  "The URL of the crc bundlepath. Defaults to the latest available version",
-			Value:  defaultBundlePath,
-			EnvVar: "VIRTUALBOX_BundlePath_URL",
+			Value:  defaultBundleName,
+			EnvVar: "VIRTUALBOX_BundleName_URL",
```

Comment:
@cfergeau I don't see any changes in the `go.mod` file, did you directly make those changes in the vendor?

Reply:
make sure you sync the gomod file when using gopath.

Reply:
now that the changes landed in machine's git, I can do the proper module updates, this should fix CI too.

Reply:
you can always use a branch (by using the SHA digest) with go.mod. As long as the branch exists on the remote repo, it should work.